### DISCUSSION TITLE
Ansible fails to SSH into VM to create template

### DISCUSSION THREAD

**Discussion by SamFCol** [vmware/packer-examples-for-vsphere#297] [Help] [answered]
Hello,
I'm pretty new to packer and the ansible environment, trying to output a rhel8 template.  I'm running into linux/rhel/8 and Ansible provisioner is unable to SSH into connect VM.

I've configured ansible.pkvars.hcl with a username and SSH public key using "ssh-keygen -t ecdsa -b 521 -C "packer"" on the packer machine. When ansible tries to SSH into the VM and fails to handshake.  Any idea how to fix this?
![packer ansible - ssh-keygen only with packer comment](https://user-images.githubusercontent.com/116681749/198029212-2c8a8a25-cd1e-4fa6-9e08-75e2f0b451ce.jpg)

https://github.com/vmware-samples/packer-examples-for-vsphere/discussions/234
answered this question. you can close.

Thank you

**Comment by tenthirtyam** [comment, marked as answer]
See #234

**Reply by tenthirtyam** [reply]
Just an FYI that I've also noted the error seen in your screenshot regarding the Ansible `command_warnings` being deprecated and will update the project with regards to that in the future.

Ryan
